### ISSUE TITLE
Req's M56D crate is old M56D with no IFF

### ISSUE THREAD

**Issue by nuklearcellphoneg** [cmss13-devs/cmss13#226]
## Description
See title

## Difference between expected and actual behavior
Should have IFF

## Steps to reproduce
Buy the crate

## Issue bingo (not required to be all checked)
- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by nuklearcellphoneg** [comment]
Here's the situation under which it occurred: 

Asked RO for M56D he orders a crate. Tosses me the setup. DS crashed into Almayer. I'm just kind of shooting down the hallway when a Doctor comes up asking why I'm shooting him and I say are you a Colonist- because I was unsure, maybe they were and got an ID?  They just said READ MY ID, so I said, whatevs look this has IFF I can't even shoot you, where I proceeded to tear him to shreds.

So that was weird, and an MP came by to stop the Doc from killing me when he got revived and this was after the round ended, so I said to the Doc naw see it has IFF I dunno why, maybe you don't have your ID on or something? Anyway, see it doesn't hurt the MP-

To which I annihilated said MP.
